api/specs/director/schemas/scripts/create_node-meta-schema.py

```diff
@@ -8,14 +8,15 @@
 from pathlib import Path
 
 import jsonref
+from common_library.json_serialization import json_dumps
 from models_library.services import ServiceMetaDataPublished
 
 CURRENT_DIR = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve().parent
 
 
 if __name__ == "__main__":
     with Path.open(CURRENT_DIR.parent / "node-meta-v0.0.1-pydantic.json", "w") as f:
-        schema = ServiceMetaDataPublished.schema_json()
+        schema = json_dumps(ServiceMetaDataPublished.model_json_schema())
```

Comment:
why is json_dumps necessary here?

Reply:
new `model_json_schema` returns a `dict` and not a `str` as the old `schema_json` would do
```
>>> a.model_json_schema()
{'properties': {'x': {'default': 4, 'title': 'X', 'type': 'integer'}}, 'title': 'A', 'type': 'object'}
```